deploy/kubernetes/overlays/windows/alpha/gcepd.yaml

```diff
@@ -0,0 +1,82 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-node
+spec:
+  template:
+    spec:
+      containers:
+        - securityContext:
+            $patch: delete
+          name: gce-pd-driver
+          image: gke.gcr.io/gcp-compute-persistent-disk-csi-driver-win
+          volumeMounts:
+            - name: kubelet-dir
+              mountPath: C:\var\lib\kubelet
+              mountPropagation: "None"
+            - name: plugin-dir
+              mountPath: C:\csi
+            - name: csi-proxy-disk-pipe
+              mountPath: \\.\pipe\csi-proxy-disk-v1alpha1
+            - name: csi-proxy-volume-pipe
+              mountPath: \\.\pipe\csi-proxy-volume-v1alpha1
+            - name: csi-proxy-filesystem-pipe
+              mountPath: \\.\pipe\csi-proxy-filesystem-v1alpha1
+            - name: kubelet-dir
+              mountPath: /var/lib/kubelet
+              mountPropagation: "Bidirectional"
+              $patch: delete
+            - name: plugin-dir
+              mountPath: /csi
+              $patch: delete
+            - name: device-dir
+              mountPath: /dev
+              $patch: delete
+            - name: udev-rules-etc
+              mountPath: /etc/udev
+              $patch: delete
+            - name: udev-rules-lib
+              mountPath: /lib/udev
+              $patch: delete
+            - name: udev-socket
+              mountPath: /run/udev
+              $patch: delete
+            - name: sys
+              mountPath: /sys
+              $patch: delete
+      nodeSelector:
```

Comment:
is there a node selector for linux that needs to be deleted? or do we need to add a linux node selector?

Reply:
no, the base set os to linux, here it will changed to windows